config/test/test.config.js

```diff
@@ -43,12 +43,9 @@ module.exports = {
         ...vendorMap,
     },
     collectCoverageFrom: [
-        "packages/**/*.js",
-        "!packages/**/types.js",
-        "!packages/**/src/**/index.js",
-        "!packages/**/dist/**/*.js",
-        "!<rootDir>/node_modules/",
-        "!packages/**/node_modules/",
+        "packages/*/src/**/*.js",
+        "packages/*/src/**/*.jsx",
```

Comment:
These files were being missed so I wasn't seeing any coverage squares in the gutter for the implementation file.